### DISCUSSION TITLE
Subject: REAME Example Missing Component Registration

### DISCUSSION THREAD

**Discussion by bestian** [f3ve/vue-markdown-it#9] [General]
Hi @f3ve,

I noticed that the example code provided in the README is missing the `components` section, which would cause an error if someone copies it as-is. Without registering the `Vue-MarkDown-It` component, it won't run successfully.

To make it work, you might want to update the example like this:

```vue
<script>
import VueMarkDownIt from 'vue-markdown-it';

export default {
  components: {
    VueMarkDownIt
  }
}
</script>
```

This addition would help others avoid potential issues when following the example.

Thanks for the awesome work!

Best regards,  
Bestian

**Comment by f3ve** [comment]
Hey thanks for the heads up I'll put a fix in for this 😄

**Comment by f3ve** [comment]
@bestian I've updated the README to include instructions on how to register the component when using [`non-setup script syntax`](https://vuejs.org/guide/components/registration.html#local-registration). Closing this as resolved. Thanks so much for your feedback!
